pkg/plugin/plugin.go

```diff
@@ -50,8 +50,9 @@ import (
 // EnvArgs args containing common, desired mac and ovs port name
 type EnvArgs struct {
 	cnitypes.CommonArgs
-	MAC     cnitypes.UnmarshallableString `json:"mac,omitempty"`
-	OvnPort cnitypes.UnmarshallableString `json:"ovnPort,omitempty"`
+	MAC         cnitypes.UnmarshallableString `json:"mac,omitempty"`
+	OvnPort     cnitypes.UnmarshallableString `json:"ovnPort,omitempty"`
+	K8S_POD_UID cnitypes.UnmarshallableString
```

Comment:
there is no assurance runtime will provide this value (altohugh for k8s we do get this information passed down by the runtime)

so in case this arg is not provided should we  store an empty value ? or just avoid adding this key to external-ids ?